### DISCUSSION TITLE
Problem receiving GitHub account verification email with Microsoft 365

### DISCUSSION THREAD

**Discussion by yamildiego** [community/community#139780] [Enterprise]
Hello,

I am having an issue when creating a GitHub account... I registered my domain and I am using Microsoft 365 Business Basic for email management. The idea is for each member to create a GitHub account using the corporate email. The problem is that the email with the code to verify the GitHub account never arrives.
I already verified my domain in the organization (adding the records)...
How can I check if the email is being sent?

Thanks!

**Comment by queenofcorgis** [comment]
Hi there @yamildiego 👋🏼 , 

Have you checked the spam folders for the emails involved? I'm not foreseeing any reason for the email not to be sent/received. As long as the domain is properly set up to receive email (you are getting other emails to these addresses), it should be working.
